File: pgm/ABC_C#/ABC_132_A.cs
using System;
using System.Collections.Generic;
// using System.Globalization;
// using System.IO;
using System.Linq;
// using System.Text;
// using System.Text.RegularExpressions

class Program
{
    static void Main()
    {
        string S = Console.ReadLine();
        HashSet<char> hs = new HashSet<char>(S);
        if (hs.Count() != 2) Console.WriteLine("No");
        else
        {
            int cnt = 0;
            foreach (char c in hs) if (S.Count(x => x == c) == 2) cnt++;
            if (cnt == 2) Console.WriteLine("Yes");
            else Console.WriteLine("No");
        }
        return;
    }
}

File: pgm/ABC_Visual_Basic/ABC_132_A.vb
' Imports System.Collections.Generic
' Imports System.Globalization
' Imports System.IO
' Imports System.Linq
' Imports System.Text
' Imports System.Text.RegularExpressions

Class Main
    Public Shared Sub Main()
        Dim S As String = Console.ReadLine()
        Dim hs As New HashSet(Of Char)(S)
        If hs.Count() <> 2
            Console.WriteLine("No")
        Else
            Dim cnt = 0
            For Each c As Char In hs
                If S.Count(function (x) x = c) = 2
                    cnt += 1
                End If
            Next
            If cnt = 2
                Console.WriteLine("Yes")
            Else
                Console.WriteLine("No")
            End If
        End If
        Return
    End Sub
End Class